app/imports/components/ActionItems/ActionItemsBoard/ActionItem.jsx

```diff
@@ -28,22 +29,39 @@ const ActionItem = ({
     <Card key={id}>
         <CardHeader
             title={!open ? `${assignee.name} [closed]` : assignee.name}
-            subtitle={closeMessage}
             avatar={assignee.avatar}
         />
 
         <CardText>
             {text} <br /> <br />
```

Comment:
You have three ways to do it better:
1. Use html tags, eg. `<p>`
2. Style it to get rid off `<br />`
3. Use both